lineman/app/components/dashboard_page/dashboard_page_controller.coffee

```diff
@@ -2,39 +2,29 @@ angular.module('loomioApp').controller 'DashboardPageController', ($rootScope, R
   $rootScope.$broadcast('currentComponent', 'dashboardPage')
   $rootScope.$broadcast('setTitle', 'Dashboard')
 
-  @loaded =
-    sort_by_date:
-      show_all: 0
-      show_unread: 0
-      show_proposals: 0
-    sort_by_group:
-      show_all: 0
-      show_unread: 0
-      show_proposals: 0
+  @loaded = {}
   @perPage =
     sort_by_date: 25
     sort_by_group: 10
-  @groupThreadCounts =
-    hidden:    0
-    collapsed: 5
-    expanded:  10
 
   @sort   = -> CurrentUser.dashboardSort
   @filter = -> CurrentUser.dashboardFilter
 
+  @loadedCount = =>
+    @loaded[@sort()] = @loaded[@sort()] or {}
+    @loaded[@sort()][@filter()] = @loaded[@sort()][@filter()] or 0
+
+  @updateLoadedCount = =>
+    current = @loadedCount()
+    @loaded[@sort()][@filter()] = current + @perPage[@sort()]
```

Comment:
(The primary thing I don't like is going to the dashboard and seeing this:

```
Today:
Thread 1
Thread 2
Thread 4

Yesterday:
Thread 5
```
Then loading more and having that thread 3 slide into 'today' unannounced; makes certain things difficult to find.